config/routes.rb

```diff
@@ -6,6 +6,8 @@
     namespace :v1 do
       resources :orders do
          get 'cart', on: :collection, to: 'orders#cart'
+         get 'placed', on: :collection, to: 'orders#get_placed'
```

Comment:
don't put the word `get` in your action name, since the HTTP request type is already GET its redundant